README.md

```diff
@@ -57,11 +57,38 @@ man f3d # Linux only
 
 # Support
 
-F3D is developed by a team of passionate devs. Please use F3D, star it on github or even become a [sponsor](https://github.com/sponsors/f3d-app) to support us!
+F3D needs you help!
 
-## Contributing
+If you can, please consider sponsoring F3D, even a small amount, it would help us offseting the maintenance cost
+as well as considering adding support for devices we do not own yet. Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).
 
-We envision F3D as a community-driven project, we love to see how the project is growing thanks to the contributions from the community. We would love to see your face in the list below! If you want to contribute to F3D, you are very welcome to! Take a look at our [contribution documentation](CONTRIBUTING.md).
+If not, please use F3D, star it on github and share the word about it!
+
+# Goals/Non-goals
+
+We believe that clearly stating the goals and non-goals of F3D would help potential users and developers figure out if F3D is the right tool for them.
+
+## Goals
+
+- Support as many 3D file formats as possible
+- Support many types of visualization
+- Support all industries dealing with 3D datasets
```

Comment:
I know what you mean but "industries" sounds a bit strong/specific; could it be reworded with something more like "use-case"/"scenario"/"workflow"/...?